Google.GenAI/Google.GenAI.csproj

```diff
@@ -40,6 +41,11 @@
     <PackageReference Include="System.Collections.Immutable" />
   </ItemGroup>
 
+  <ItemGroup Condition="'$(TargetFramework)' == 'netstandard2.0'">
+    <PackageReference Include="System.Text.Json" />
+    <PackageReference Include="System.Collections.Immutable" />
```

Comment:
I'd suggest just dropping the S.C.Immutable reference. All the uses of it I see are erroneous, using ImmutableDictionary where just a normal Dictionary would suffice and be better.